online_meetings/canvas_zoom_meetings.py

```diff
@@ -102,13 +103,10 @@ def get_zoom_details(self, url: str, data: Dict[str, str], course_id: int):
                         self.zoom_courses_meetings.append({
                             'course_id': course_id,
                             'meeting_id': meeting['meetingId'],
-                            'meeting_number': meeting['meetingNumber'],
                             'host_id': meeting['hostId'],
                             'topic': meeting['topic'],
```

Comment:
Do I either of these need to be dropped as well? I have not kept up on what is been approved for storing/needed by IQ. Looking back at issue #100, it looks like at least `topic` was supposed to be dropped, but things may have changed.

Reply:
Thanks for double checking, yeah they wanted the topic removed. The host id is an alphanumeric id (rather than something that can be identified) and I think this was fine to leave in there.